Examples/ArcGISToolkitExamples.xcodeproj/project.pbxproj

```diff
@@ -547,6 +473,17 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCSwiftPackageProductDependency section */
+		79CA564325F1952D00E9FE3D /* ArcGISToolkit */ = {
+			isa = XCSwiftPackageProductDependency;
+			productName = ArcGISToolkit;
+		};
+		79CA564525F1953500E9FE3D /* ArcGISToolkit */ = {
+			isa = XCSwiftPackageProductDependency;
+			productName = ArcGISToolkit;
+		};
```

Comment:
This duplicate identical dependency is caused by the library being explicitly added to the Dependencies build phase:

![image](https://user-images.githubusercontent.com/2257493/110380145-90121880-800c-11eb-84b3-b05c089bc5b9.png)

Xcode seems to implicitly track Swift package dependencies, so the explicit dependency could be removed, eliminating this duplicate entry here.

Reply:
Yep let me fix that